aws-api/src/main/java/com/amplifyframework/api/aws/AppSyncPage.java

```diff
@@ -0,0 +1,33 @@
+package com.amplifyframework.api.aws;
+
+import androidx.annotation.NonNull;
+
+import com.amplifyframework.api.graphql.GraphQLRequest;
+import com.amplifyframework.api.graphql.GraphQLResponse;
+import com.amplifyframework.api.graphql.Page;
+
+final class AppSyncPage<M> extends Page<M> {
```

Comment:
Looks reasonable, remember to add `equals`, `hashCode`, `toString` before pushing

Reply:
Added in latest commit!